recipes/_install.rb

```diff
@@ -29,16 +29,16 @@ def nsp
 include_recipe 'cerner_splunk::_restart_marker'
 
 # Actions
-# This service definition is used only for ensuring splunk is started during the run
-service 'splunk-start' do
+# This service definition is used for ensuring splunk is started during the run and to stop splunk service
+service 'splunk' do
   service_name service
   action :nothing
-  supports status: true, start: true
+  supports status: true, start: true, stop: true
   notifies :delete, 'file[splunk-marker]', :immediately
```

Comment:
Splunk is stopped after the member is removed from the SHC to avoid messages regarding its removal from appearing on the captain.
There are couple of more commands that need to be executed before you bring that instance up  which we are not supporting now.
http://docs.splunk.com/Documentation/Splunk/6.4.1/DistSearch/Removeaclustermember

I had a splunk-stop service before, which was removed after discussing about it in this [comment](https://github.com/cerner/cerner_splunk/pull/124#discussion_r68251562). I can add it back to make the change passive if @gravesb feels good about it.